string_to_string_test.go

```diff
@@ -140,13 +140,14 @@ func TestS2SCalledTwice(t *testing.T) {
 	var s2s map[string]string
 	f := setUpS2SFlagSet(&s2s)
 
-	in := []string{"a=1,b=2", "b=3", `"e=5,6"`, `f="7,8"`}
```

Comment:
Can you add a check that `len(s2s) == len(expected)` so we can't make this mistake again?

Reply:
Added this check. PTAL